containers/codespaces-linux/.devcontainer/library-scripts/jupyterlab-debian.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
See the readme in this folder - anything here that isn't in `/library-scripts` will be deleted with newer copies from that directory. 

If we're not taking this the whole way and authoring a feature, I guess we might as well just add that statement as a line in the dockerfile itself?